packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+import ActionHeaderContainer from './_ActionHeaderContainer';
+import styles from './ActionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Text to be displayed as the title in the header bar.
+   */
+  title: PropTypes.string,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. Default `level=1`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
+  /**
+   * Callback function for when the close button is clicked. The back button will not display if this is not set.
```

Comment:
We probably should clean up the documentation for the onClose, onBack and keepCloseButton features. Let me see what suggestion I can help with, but it feels like you have to do a scavenger hunt to determine if you will see a back button or not

Reply:
Glad you caught this. I removed the text "The back button will not display if this is not set." as it was inaccurate [here](https://github.com/cerner/terra-core/commit/5b230102a59e8b12ddab3baf4971a109535b0884).